templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/service/impl/config/WebConfig.java

```diff
@@ -59,6 +60,7 @@ public FilterRegistrationBean diagnosticContextFilter() {
     this.beanFactory.autowireBean(diagnosticContextFilter);
     registration.setFilter(diagnosticContextFilter);
     registration.addUrlPatterns(ServiceConstants.URL_PATH_SERVICES + "/*");
+    registration.setOrder(1);
```

Comment:
these two should be the actual fix. A nicer option is to use "add filter before/after" mechanism of spring.
How did you test this?

Reply:
As discussed already this add filter before/after is only available for spring-security filter chain. We do not want to couple with spring-security here so this fix should be fine.